src_server/package.json

```diff
@@ -14,6 +14,7 @@
     "fs-extra": "^10.0.0",
     "is-port-reachable": "^3.0.0",
     "morgan": "^1.10.0",
+    "n": "^7.5.0",
```

Comment:
this package should not be included in our package.json. Upgrading of nodejs should be included in our system installation.